lib/driver.js

```diff
@@ -560,6 +561,8 @@ class AndroidUiautomator2Driver extends BaseDriver {
     } else {
       this.jwpProxyAvoid = NO_PROXY;
     }
+    this.jwpProxyAvoid = this.jwpProxyAvoid.concat(this.additionalJwpProxyAvoid);
```

Comment:
I think we don't need `this.additionalJwpProxyAvoid` variable. Just change this line to

```js
if (this.opts.nativeWebScreenshot) {
   this.jwpProxyAvoid = [...this.jwpProxyAvoid, ['GET', new RegExp('^/session/[^/]+/screenshot')]];
}
```

Reply:
Oh, I didn't realize `this.opts` is still available :D. Let me rectify.